runtime/development/src/lib.rs

```diff
@@ -1914,6 +1917,24 @@ impl_runtime_apis! {
 		}
 	}
 
+	// RewardsApi
+	impl runtime_common::apis::RewardsApi<Block, AccountId, Balance, RewardDomain, CurrencyId> for Runtime {
+		fn list_currencies(account_id: AccountId) -> Option<Vec<(RewardDomain, CurrencyId)>> {
+			let currencies = pallet_rewards::Pallet::<Runtime, pallet_rewards::Instance1>::list_currencies(account_id);
+
+			Some(currencies)
+		}
+
+		fn compute_reward(reward_currency_id: (RewardDomain, CurrencyId), account_id: AccountId) -> Option<Balance> {
+			let reward = <pallet_rewards::Pallet::<Runtime, pallet_rewards::Instance1> as AccountRewards<AccountId>>::compute_reward(reward_currency_id, &account_id);
+
+			match reward {
+				Ok(balance) => Some(balance),
+				Err(_) => None,
+			}
```

Comment:
```suggestion
		<pallet_rewards::Pallet::<Runtime, pallet_rewards::Instance1> as AccountRewards<AccountId>>::compute_reward(reward_currency_id, &account_id).ok()
```

You can call `.ok()` on a `Result` to convert it to `Option` 👍

Reply:
Will change, thank you!